components/AppLayout/UpgradeDialog.js

```diff
@@ -15,95 +15,41 @@ import LEVEL_NAMES from 'constants/levelNames';
 
 import upgradeImage from './images/upgrade.png';
 
-const getConicGradient = (color1, color2, startAngle) => {
-  return `repeating-conic-gradient( 
-      from ${startAngle}deg, 
-      ${color1} ${startAngle}deg ${startAngle + 15}deg, 
-      ${color2} ${startAngle + 15}deg ${startAngle + 30}deg
-)`;
+const getSpinKeyframes = (steps, startDeg, color1, color2) => {
+  const keyframes = {};
+  const progressUnit = 100 / steps;
+  const degUnit = 360 / steps;
```

Comment:
Since the pattern repeats every 30 degree and it is an repeating animation, we can just animate from 0deg to 30deg and have the same visual effect.

Reply:
Updated with `30deg / steps`.
With the same 10 steps, the animation became smoother because the increase of density:
![spinner](https://user-images.githubusercontent.com/108608/103478577-a0d2c200-4e02-11eb-9a3b-c5806de8d85f.gif)